source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,263 @@
+installing packages individually.
+
+Executing `bower install` will install all of the dependencies listed in
+`bower.json` in one step.
+
+Ember CLI is configured to have git ignore your `bower_components` directory by
+default. Using the Bower configuration file allows collaborators to fork your
+repo and get their dependencies installed locally by executing
+`bower install` themselves.
+
+Ember CLI watches `bower.json` for changes. Thus it reloads your app if you
+install new dependencies via `bower install <dependencies> --save`.
+
+Further documentation about Bower is available at their
+[official documentation page](http://bower.io/).
+
+### Compiling Assets
+
+In your `Brocfile.js` specify a dependency before calling `app.toTree()`. You
+can only import assets that are within the `bower_components` or `vendor`
+directories. The following example scenarios illustrate how this works.
+
+#### Javascript Assets
+
+##### Standard Non-AMD Asset
+
+Provide the asset path as the first and only argument:
+
+```Brocfile.js
+app.import('bower_components/moment/moment.js');
+```
+
+From here you would use the package as specified by it's documentation, usually a global variable.
+In this case it would be:
+
+```js
+import Ember from 'ember';
+/* global moment */
+// No import for moment, it's a global called `moment`
+
+// ...
+var day = moment('Dec 25, 1995');
+```
+
+_Note: Don't forget to make JSHint happy by adding a `/* global MY_GLOBAL */` to your module, or
+by defining it within the `predefs` section of your `.jshintrc` file._
+
+##### Standard AMD Asset
+
+Provide the asset path as the first argument, and the list of modules and exports as the second:
+
+```Brocfile.js
+app.import('bower_components/ic-ajax/dist/named-amd/main.js', {
+  exports: {
+    'ic-ajax': [
+      'default',
+      'defineFixture',
+      'lookupFixture',
+      'raw',
+      'request',
+    ]
+  }
+});
+```
+
+To use this asset in your app, import it.
+For example, with `ic-ajax`, when to use `ic.ajax.raw`:
+
+```js
+import { raw as icAjaxRaw } from 'ic-ajax';
+//...
+icAjaxRaw( /* ... */ );
+```
+
+##### Environment Specific Assets
+
+If you need to use different assets in different environments, specify an object as the first parameter. That object's key should be the environment name, and the value should be the asset to use in that environment.
+
+```Brocfile.js
+app.import({
+  development: 'bower_components/ember/ember.js',
+  production:  'bower_components/ember/ember.prod.js'
+});
+```
+
+If you need to import an asset in one environment but not import it or any alternatives in other environments then you can wrap `app.import` in an `if` statement.
+
+```Brocfile.js
+if (app.env === 'development') {
+  app.import('vendor/ember-renderspeed/ember-renderspeed.js');
+}
+```
+
+##### Customizing a built-in Asset
+
+This is somewhat non-standard and discouraged, but suppose that due to a requirement in your application that you need to use the full version of
```

Comment:
:fire:. can we come up with a "standard and encouraged" example instead? ;P cc @michaelrkn